tools/cockpit.spec

```diff
@@ -327,7 +327,7 @@ The Cockpit components for managing software updates for ostree based systems.
 
 %package pcp
 Summary: Cockpit PCP integration
-Requires: %{name}-bridge = %{version}-%{release}
+Requires: %{name}-bridge > 134
```

Comment:
I don't think we should artificially bump the requirements in manifest.json - only if the actual content demands it. Anything else is packaging magic/glue on top.